docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -411,16 +202,15 @@ Event Hubs Client Configuration Options`) can be applied for each specific consu
 
 |*spring.cloud.stream.eventhubs.bindings.<binding-name>.producer*.sync
 | boolean
-|The switch flag for sync of producer
+|The switch flag for sync of producer. If true, the producer will wait for a response after a send operation.
 
 |*spring.cloud.stream.eventhubs.bindings.<binding-name>.producer*.send-timeout
-| long
-| The timeout value for sending of producer
+| Duration
+|The amount of time to wait for a response after a send operation. Will take effect only when a sync producer is enabled.
 |===
 
 ====== Advanced Producer Configuration
-The configuration in the above first 2 sections(`Azure Common Configuration Options`, `Azure
-Event Hubs Client Configuration Options`) can be applied for each specific producer by replacing the prefix of *spring.cloud.azure.eventhubs* with *spring.cloud.stream.eventhubs.bindings.<binding-name>.producer*.
+The above <<eventhubs-connection-configration, connection>> and <<Configuration.adoc#configuration, common Azure SDK client>> configuration are supported to be customized for each binder producer, which can be configured with the prefix *spring.cloud.stream.eventhubs.bindings.<binding-name>.producer.*.
```

Comment:
*spring.cloud.stream.eventhubs.bindings.<binding-name>.producer.* to `spring.cloud.stream.eventhubs.bindings.<binding-name>.producer.`